install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,181 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[renumbering-sdn-networks]]
+== Renumbering the SDN Network
+
+It is difficult to change the networking configuration of an
+{product-title} cluster after it has been deployed, but if absolutely
+necessary, it can be done.
+
+=== Expanding the Cluster IP Range By Adding Nodes
+
+The easiest reconfiguration involves changing the
+`*clusterNetworkCIDR*` to make room to add additional nodes, while
+leaving existing Nodes and Pods untouched.
+
+For example, with the default `*clusterNetworkCIDR*` value of
+*10.128.0.0/14* and the default `*hostSubnetLength*` of *9*, you can
+have 512 nodes, with Pod subnets from *10.128.0.0/23* to
+*10.131.254.0/23*. (Older versions of {product-title} had smaller
+defaults.) If you changed `*clusterNetworkCIDR*` to *10.128.0.0/13*,
+then you could have an additional 512 nodes, with Pod IPs going up to
+*10.135.255.255*.
+
+To do this, simply change the `*clusterNetworkCIDR*` value in the
+`*networkConfig*` stanza of the *_master-config.yaml_* file, and then
+restart the master; for single master clusters:
+
+----
+# systemctl restart atomic-openshift-master
+----
+
+For multi-master clusters:
+
+----
+# systemctl restart atomic-openshift-master-controllers
+----
+
+You will also need to restart the node service on the master and on
+each of the nodes, to make them update their IP routing information to
+take the larger cluster network into account:
+
+----
+# systemctl restart atomic-openshift-node
+----
+
+Note that this only works if you are changing `*clusterNetworkCIDR*`
+to a new value which is larger than the old value, and which includes
+the old value as a subset; if you try to change it to any other value,
+then when you restart the master, it will complain about the mismatch
+and refuse to start. You will need to use the procedure below in this
+case.
```

Comment:
Should we mention that additional nodes can be created and point to how to do it?